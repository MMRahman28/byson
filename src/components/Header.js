import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Poems</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>PoemsPage</NavLink>
        <NavLink to="/create" activeClassName="is-active">Create poem</NavLink>

        <NavLink to="/help" activeClassName="is-active">Help</NavLink>

    </header>
);
export default Header;
