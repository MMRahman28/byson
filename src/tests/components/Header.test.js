import { shallow } from 'enzyme';
//import toJSON from 'enzyme-to-json';
import React from 'react';
import Header from '../../components/Header';

test('should render Header correctly', () => {
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header />);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
  const wrapper = shallow(<Header />);
  //expect(wrapper.find('h1').text()).toBe('Poems');
  expect(wrapper).toMatchSnapshot();
  });
