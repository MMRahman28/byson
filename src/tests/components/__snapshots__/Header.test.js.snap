// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Poems
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    PoemsPage
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
    Create poem
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
